deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -218,55 +288,68 @@ Resources:
               - 'glue:*'
             Resource: '*'
 
-          - Sid: EC2
-            Effect: Allow
-            Action:
-              - 'ec2:Describe*'
-              - 'ec2:*SecurityGroup*'
-              - 'ec2:Create*'
-              - 'ec2:*InternetGateway*'
-              - 'ec2:Associate*'
-              - 'ec2:Disassociate*'
-              - 'ec2:Delete*'
-              - 'ec2:Modify*'
-              - 'ec2:Get*'
-              - 'ec2:*Address'
-              - 'elasticfilesystem:CreateFileSystem'
-              - 'elasticfilesystem:TagResource'
-              - 'elasticfilesystem:UntagResource'
-              - 'elasticfilesystem:DeleteFileSystem'
-            Resource: '*'
+          - !If 
+            - MLStudioEnabledCondition
```

Comment:
Found some errors when creating Notebooks needing some additional EC2 permissions

The quick fix is to switch `MLStudioEnabledCondition` with `SagemakerComponentsEnabledCondition` (tested and all works then)

We could theoretically also determine the precise permissions needed for each Notebook and MLStudio (MLStudio permissions likely being a superset of the notebook permissions plus others) but it depends on the level of granularity we are wanting here

I am good to just pass `SagemakerComponentsEnabledCondition` unless others disagree